test/jdk/sun/security/lib/cacerts/VerifyCACerts.java

```diff
@@ -47,12 +47,12 @@ public class VerifyCACerts {
             + File.separator + "security" + File.separator + "cacerts";
 
     // The numbers of certs now.
-    private static final int COUNT = 111;
+    private static final int COUNT = 109;
 
     // SHA-256 of cacerts, can be generated with
     // shasum -a 256 cacerts | sed -e 's/../&:/g' | tr '[:lower:]' '[:upper:]' | cut -c1-95
     private static final String CHECKSUM
-            = "B7:60:5A:7A:01:0A:9F:47:E3:46:B9:30:E1:FC:A2:71:69:58:76:CB:8C:85:2B:FF:1A:D5:92:71:AF:F5:60:8F";
+            = "07:21:E0:F8:EA:55:CC:93:24:2E:74:07:4B:6B:CE:F3:81:C3:BB:47:5B:85:A2:F1:9E:44:CD:C0:99:55:D7:5F";
```

Comment:
Strange that the CHECKSUM is different in 17u-dev before and after the change , compared to e.g. 21 and head ; but the number of certs (111 -> 109) seems to be the same.
Is there some other backport coming/missing ?

Reply:
Hi Matthias, this is always the case with changes to the certificates in 17. See also previous changes
https://github.com/openjdk/jdk17u-dev/commit/4e67796c3fcbffa706be8fabb56d00f9128edac1
or others in this list: https://github.com/openjdk/jdk17u-dev/commits/4e67796c3fcbffa706be8fabb56d00f9128edac1/test/jdk/sun/security/lib/cacerts/VerifyCACerts.java
I think this is caused by the different location and build functionality used in 17.